content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+MHCflurry [@doi:10.1101/054775] transforms peptide sequences using a NetMHC-type length adjustment before encoding each amino acid using a corresponding learned vector representation.
+In training the MHCflurry feed-forward neural network, the authors impute missing MHC-peptide binding affinities using a Gibbs sampling method, showing that imputation improves performance for data-sets with roughly 100 or fewer training examples.
+MHCflurry's imputation method increases its performance on poorly characterized alleles, making it competitive with NetMHCpan for this task.
+
+Using existing methods to represent uniform-length peptides, deep and mixed approaches have also shown promise towards prediction.
+Kuksa et al. [@doi:10.1093/bioinformatics/btv371] pre-trained a higher-order neural network (HONN) using a semi-restricted Boltzmann machine and compared performance with a deep neural network (DNN).
+They also propose a higher-order SVM, which, when paired with the HONN showed superior performance, which exceeded that of the DNN.
+The authors use Blosum encoding for 9-mer peptides.
+Vang et al. [@doi:10.1093/bioinformatics/btx264] combined deep learning with tools from natural language processing to approach 9-mer peptides.
+They employ a technique called distributed representation to map amino acids onto a 15-dimensional vector space based on their context relation to other amino acids.
+A convolutional neural network, titled HLA-CNN, takes peptide sequences and replaces individual amino acids with their corresponding vectors to predict binding.
+
+Deep learning's unique flexibility was recently leveraged by Bhattacharya et al., who used a gated recurrent neural network method called MHCnuggets to overcome the difficulty of multiple length peptides.
+Under this framework, they used smoothed sparse encoding to represent amino acids individually.
+Because MHCnuggets had to be trained for every MHC allele, performance was far better for alleles with abundant, balanced training data.
+
+In a comparison of several current methods, Bhattacharya et al. found that the top methods — NetMHC, NetMHCpan, MHCflurry, and MHCnuggets — showed comparable performance, but large differences in speed.
+Convolutional neural networks (in this case, HLA-CNN) showed comparatively poor performance, while shallow and recurrent neural networks performed the best.
+They found that MHCnuggets — the recurrent neural network — was by far the fastest training among the top performing methods.
+In predicting interactions between proteins, deep learning has achieved state-of-the-art results and shows promise to overcome previous challenges in the field.
+
+### PPI networks and graph analysis
+
+The abundance of large networks of biological data presents a different approach to computational prediction.
```

Comment:
I think we should lead with something about graph structure and how it provides important information about biological networks but cannot be modeled directly with standard neural networks.  Would you agree with that?